pkg/printers/internalversion/printers_test.go

```diff
@@ -257,8 +257,12 @@ func testPrinter(t *testing.T, printer printers.ResourcePrinter, unmarshalFunc f
 	}
 }
 
+func yamlUnmarshal(data []byte, v interface{}) error {
+	return yaml.Unmarshal(data, v)
```

Comment:
what changed here in the type?

Reply:
`func Unmarshal(y []byte, o interface{}, opts ...JSONOpt) error` is the new method signature. please see vendor/github.com/ghodss/yaml/yaml.go